k8s/app.yml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
```

Comment:
Is this file needed as we want to start it from the `operation` repo?

Reply:
Yeah, I think we do not need it here, will remove it